gnome-shell/src/meson.build

```diff
@@ -96,6 +106,10 @@ foreach variant: variants
   if is_variant
     subdir(variant)
   endif
+  
```

Comment:
I suspect this two `if` blocks (between 106 and 112) are not necessary anymore. `is_variant` is true if `variant != light` right? So:

- if `is_variant` is true, then `variant == 'light'` and the subdir command is equal to `subdir('light')`
- if `is_variant` is false, then `variant != 'light'`, but subdir command is still equal to `subdir('light)`
 
 I believe this was necessary when the variantes were three (default, light and dark).